src/clojure_lsp/shared.clj

```diff
@@ -1,14 +1,37 @@
 (ns clojure-lsp.shared
   (:require
-   [clojure.string :as string])
+   [clojure.java.shell :as shell]
+   [clojure.string :as string]
+   [clojure.tools.logging :as log])
   (:import
    [java.net URI]
    [java.nio.file Paths]
    [org.eclipse.lsp4j Range]))
 
-(defn windows-os? []
+(def windows-os?
```

Comment:
I changed this because I don't think the OS can ever change mid jvm-run, but I'll change it back if you want. Just prevents a probably very tiny/insignificant amount of repeated work.

Reply:
It's better this way, nice!